src/modules/map/components/LocationPopupPosition.vue

```diff
@@ -0,0 +1,192 @@
+
+import proj4 from 'proj4'
+import { computed, onMounted, ref, toRefs, watch } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+import { requestHeight } from '@/api/height.api'
+import { registerWhat3WordsLocation } from '@/api/what3words.api'
+import LocationPopupCopySlot from '@/modules/map/components/LocationPopupCopySlot.vue'
+import {
+    LV03Format,
+    LV95Format,
+    MGRSFormat,
+    UTMFormat,
+    WGS84Format,
+} from '@/utils/coordinates/coordinateFormat'
+import { WGS84 } from '@/utils/coordinates/coordinateSystems'
+import log from '@/utils/logging'
+
+const props = defineProps({
+    coordinate: {
+        type: Boolean,
+        required: true,
+    },
+    clickInfo: {
+        type: Object,
+        required: true,
+    },
+    projection: {
+        type: Object,
+        required: true,
+    },
+    currentLang: {
+        type: Object,
+        required: true,
+    },
+})
+const { coordinate, clickInfo, projection, currentLang } = toRefs(props)
+
+const what3Words = ref(null)
+const height = ref(null)
+
+const i18n = useI18n()
+
+const coordinateWGS84Metric = computed(() => {
+    return proj4(projection.value.epsg, WGS84.epsg, coordinate.value)
+})
+const coordinateWGS84Plain = computed(() => {
+    // we want to output lat / lon, meaning we have to give the coordinate as y / x
+    return coordinateWGS84Metric.value
+        .slice()
+        .reverse()
+        .map((val) => WGS84.roundCoordinateValue(val).toFixed(6))
+        .join(', ')
+})
+const heightInFeet = computed(() => {
+    if (height.value?.heightInFeet) {
+        return `${height.value.heightInFeet}ft`
+    }
+    return null
+})
+const heightInMeter = computed(() => {
+    if (height.value?.heightInMeter) {
+        return `${height.value.heightInMeter}m`
+    }
+    return ''
+})
+
+onMounted(() => {
+    if (clickInfo.value) {
+        updateWhat3Word()
+        updateHeight()
+    }
+})
+
+watch(clickInfo, (newClickInfo) => {
+    if (newClickInfo) {
+        updateWhat3Word()
+        updateHeight()
+    }
+})
+watch(currentLang, () => {
+    updateWhat3Word()
+})
+
+async function updateWhat3Word() {
+    try {
+        what3Words.value = await registerWhat3WordsLocation(
+            coordinate.value,
+            projection.value,
+            currentLang.value
+        )
+    } catch (error) {
+        log.error(`Failed to retrieve What3Words Location`, error)
+        what3Words.value = null
+    }
+}
+async function updateHeight() {
+    try {
+        height.value = await requestHeight(coordinate.value, projection.value)
+    } catch (error) {
+        log.error(`Failed to get position height`, error)
+        height.value = null
+    }
+}
+</script>
+
+<template>
+    <div
+        id="nav-local"
+        class="tab-pane fade"
+        role="tabpanel"
+        aria-labelledby="nav-local-tab"
+        data-cy="import-file-local-content"
+    >
+        <form class="input-group rounded needs-validation">
```

Comment:
Remove the `input-group need-validations` classes this is only needed for from. Also don't use a form but a simple div here. Check if `rounded` class is needed but probably not

Reply:
You might not need at all this html element as you already have a div below